### DISCUSSION TITLE
[WindowsAppSdk][WASM][.NET 6] What to deploy and how?

### DISCUSSION THREAD

**Discussion by DierkDroth** [unoplatform/uno#7963] [Q&A] [answered]
I read through [this](https://platform.uno/blog/introduction-to-webassembly-for-net-developers-building-with-uno-platform-xaml-and-c/), however it's still not clear to me how I would deploy my WASM app to a random web space (e.g. on AWS S3).

- I ran 'msbuild /t:publish ...' on the WASM solution
- I take it I needed to look into the directory ./bin/Release/.net6.0/publish. Is that correct?
- What from within that directory should I upload (please see screenshot below)?
- Which subdirectories are required? **Both** the Brotli **and** the GZ directory? The unzipped 'package_*' directory as well?
- Is there anything else I needed to do? Or am I ready to go by just starting 'index.html' in browser?

Thanks in advance

![image](https://user-images.githubusercontent.com/26179242/151971032-345e4a98-d466-4956-aee7-ed3cdf989897.png)

**Comment by jeromelaban** [comment, marked as answer]
We do not have the directions for hosting on all types of services, but here's [some documentation to get you going](https://platform.uno/docs/articles/how-to-host-a-webassembly-app.html).

All files in the dist folder are required for publishing an Uno app.

**Reply by DierkDroth** [reply]
Thanks for those directions @jeromelaban

**Reply by rotemleibo** [reply]
@jeromelaban  Cam i run it on Kestrel web server?
